src/components/annual-report/ReportPrintHeader.js

```diff
@@ -0,0 +1,64 @@
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { useConfigurations } from '../../utils/index.js'
+import { CORE_GROUP_CODE } from '../report-parameter-selector/index.js'
+import styles from './ReportPrintHeader.module.css'
+
+/** Shown only in print to indicate report parameters */
+export const ReportPrintHeader = ({ reportParameters }) => {
+    const configurations = useConfigurations()
+
+    const groupName = React.useMemo(() => {
+        if (!reportParameters?.groupID) {
+            return null
+        }
+        if (reportParameters.groupID === CORE_GROUP_CODE) {
+            return i18n.t('[Core]')
+        }
+        const group = configurations.groups.find(
+            (group) => group.code === reportParameters.groupID
+        )
+        return group.name
+    }, [configurations, reportParameters])
+
+    if (!reportParameters || Object.keys(reportParameters).length === 0) {
+        return null
+    }
+
+    const { orgUnitName, orgUnitLevelName, periods } = reportParameters
+
+    return (
+        <div className={styles.showOnlyInPrint}>
+            <h1 className={styles.title}>
+                {i18n.t('Data Quality Annual Report')}
+            </h1>
+            <div className={styles.parametersContainer}>
+                <p className={styles.parameterLine}>
+                    {/**
+                     * Reminder: ':' is the namespace separater for i18next so
+                     * this needs interpolation:
+                     */}
+                    <strong>{i18n.t('Data{{c}} ', { c: ':' })}</strong>
```

Comment:
you can also override the separator in the options you pass to the translation function, e.g.
https://github.com/hisprwanda/who-data-quality-annual-report/blob/main/src/components/annual-report/common/Warnings.js#L73-L79
docs: https://www.i18next.com/overview/configuration-options

Reply:
Ah nice, thank you! I'll use that 🙌